ext/cl_khr_command_buffer_mutable_dispatch.asciidoc

```diff
@@ -254,6 +259,9 @@ CL_MUTABLE_COMMAND_COMMAND_TYPE_KHR                0x12AD
 
 // Bits for cl_command_buffer_flags_khr
 CL_COMMAND_BUFFER_MUTABLE_KHR                    (0x1 << 1)
+
+// Bits for cl_mutable_dispatch_promises_khr bitfield
+CL_MUTABLE_DISPATCH_PROMISE_NO_ADDITIONAL_WORK_GROUPS_KHR	(0x1 << 0)
```

Comment:
Should also be added to XML file
```
 <enums name="cl_mutable_dispatch_promises_khr" vendor="Khronos" type="bitmask">
        <enum bitpos="0"            name="CL_MUTABLE_DISPATCH_PROMISE_NO_ADDITIONAL_WORK_GROUPS_KHR"/>
         <unused start="1" end="31"/>
 </enums>
```

The XML definition of the extension should also be updated to reflect these new XML additions https://github.com/KhronosGroup/OpenCL-Docs/blob/c37e00f1423379a882c173c12ed1aa5ca9d8e51d/xml/cl.xml#L7268

Reply:
CL_MUTABLE_DISPATCH_PROMISE_NO_ADDITIONAL_WORK_GROUPS_KHR is defined in line 1348.